features/Screen-Space Shadows/Shaders/ScreenSpaceShadows/RaymarchCS.hlsl

```diff
@@ -94,10 +56,10 @@ float ScreenSpaceShadowsUV(float2 texcoord, float3 lightDirectionVS)
 	float stepLength = maxDistance / (float)maxSteps;
 
 	// Compute ray step
-	float3 rayStep = lightDirectionVS * stepLength;
+	float3 rayStep = InvDirLightDirectionVS.xyz * stepLength;
```

Comment:
This seems like a pretty big change if you're doing the inverse. Does it break flat?

Reply:
I only removed the input to the function, was only used once. But I removed that now to make less changes on this PR